dutch/over-het-hek/flippo/scheer/scheer.md

```diff
@@ -0,0 +1 @@
+De flippo gaat best hard, even waan je jezelf een olympisch atleet, maar snel bezef je dat het hier maar om een stuk karton half zo groot als een bierviltje gaat. Even was het leuk. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.